src/Components/Projects/Projects.tsx

```diff
@@ -30,107 +43,130 @@ const handleClose = () => setModalState("close");
-      </thead>
-
-      <tbody className='text-center '>
-        {allProjs.length>0&&allProjs.map((pro)=>(<tr key={pro?.id}>
-          <td>{pro?.title}</td>
-          <td className='text-white' ><div className='status'>{pro?.manager?.userName}</div></td>
-          <td>{pro?.description}</td>
-          <td>
-          <div className="img-container">
-
-      {pro?.imagePath?(<img className='img-fluid' src={`https://upskilling-egypt.com/`+pro?.imagePath} />):(<img className='img-fluid'src={avatar}/>)}
+        <div className="col-md-12 ">
+          <input onChange={getTitleValue} type="text" placeholder='    SearchFleets ' className='rounded-4 border-1 mb-4 pro' />
+          <Table striped bordered hover>
+            <thead >
+              <tr className='bg text-center' >
+                <th>Title</th>
+                <th>Manager</th>
+                <th>Description</th>
+                <th>Image</th>
+                <th>Date Created</th>
+                <th>
+                </th>
+              </tr>
+
+
+            </thead>
 
+            <tbody className='text-center '>
+              {allProjs.length > 0 && allProjs.map((pro) => (<tr key={pro?.id}>
+                <td>{pro?.title}</td>
+                <td className='text-white' ><div className='status'>{pro?.manager?.userName}</div></td>
+                <td>{pro?.description}</td>
+                <td>
+                  <div className="img-container">
+
+                    {pro?.imagePath ? (<img className='img-fluid' src={`https://upskilling-egypt.com/` + pro?.imagePath} />) : (<img className='img-fluid' src={avatar} />)}
+
+                  </div>
+                </td>
+                <td className='datepicker' >{pro?.creationDate}</td>
+                <td> <div className="dropdown ">
+                  <button className="btn btn-transparent dropdown-toggle " type="button" data-bs-toggle="dropdown" aria-expanded="false">
+                    <i className="fa-solid fa-ellipsis-vertical"></i>
+                  </button>
+                  <ul className="dropdown-menu ">
+                    <li onClick={() => viewPro(pro?.id)}><a className="dropdown-item" > <i className=" fa-regular fa-eye "></i> View</a></li>
+                    <li onClick={() => editShow(pro?.id)} ><a className="dropdown-item " > <i className="fa-regular fa-pen-to-square pe-2"></i>Edit</a></li>
+                    <li onClick={() => showDeleteModel(pro?.id)}><a className="dropdown-item" > <i className="fa-solid fa-trash pe-2"></i>Delete</a></li>
+                  </ul>
+                </div></td>
+              </tr>
+              ))}
+
+
+
+            </tbody>
+
+          </Table>
+          <nav aria-label="..." className='me-5'>
+            <ul className="pagination pagination-xs  w-50 ms-auto d-flex justify-content-end">
+              {pagesArray.map((pageNo) => <li className="page-item  "><a onClick={() => showAllProjects(pageNo, searchInput)} className="page-link">{pageNo}</a></li>
```

Comment:
Here it is better to use button instead of anchor for the onClick, because anchors elements are meant to be used for navigation, but here we are not storing the pagination in the URL by something like by using useSearchParams and setting them in the URL.